index.bs

```diff
@@ -2334,6 +2309,121 @@ The [=default allowlist=] for this feature is <code>["self"]</code>.
 
 Note: If the document's origin is not allowed to use the <code>"xr-spatial-tracking"</code> feature policy any [=immersive sessions=] will be blocked, because all [=immersive sessions=] require some use of spatial tracking. [=Inline sessions=] will still be allowed, but restricted to only using the {{XRReferenceSpaceType/"viewer"}} {{XRReferenceSpace}}.
 
+
+Permissions API Integration {#permissions}
+---------
+
+The [[!permissions]] API provides a uniform way for websites to request permissions from users and query which permissions they have been granted. 
+
+The <dfn enum-value for="PermissionName">"xr"</dfn> [=powerful feature=]’s permission-related algorithms and types are defined as follows: 
+
+
+<dl>
+<dt>[=permission descriptor type=]</dt>
+<dd>
+
+<pre class="idl">
+dictionary XRPermissionDescriptor: PermissionDescriptor {
+  XRSession session;
+  sequence&lt;any&gt; requiredFeatures;
+  sequence&lt;any&gt; optionalFeatures;
+};
+</pre>
+
+{{PermissionDescriptor/name}} for {{XRPermissionDescriptor}} is {{PermissionName/"xr"}}.
+
+
+</dd>
+
+<dt>[=permission result type=]</dt>
+<dd>
+<pre class=idl>
+dictionary XRPermissionStatus: PermissionStatus {
+  sequence&lt;any&gt; granted;
```

Comment:
nit: I would call this `features`, not `granted` because it's hard to understand what that is without a lot of context. It kind of relates to #860 too.

Reply:
Personally I find `granted` clearer than `features`. But open to changing if others feel differently.

Reply:
cc @toji opinions on naming?